### PR TITLE
fix/i18n_translation_utf8

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Fix utf8 encoding in templates and saving files

Impacts
=======
The portal, admin, email templates

Delete branch after merge
=========================
NO

UPGRADE file entries
====================

We had an issue where configuration we being saved as latin1 instead of utf8.

This script will convert all latin1 config file to utf8

addons/upgrade/to-8.3-conf-latin1-to-utf8.sh

### CODE REVIEW

File: addons/upgrade/to-8.3-conf-latin1-to-utf8.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+find /usr/local/pf/conf -type f -print0 | while read -d $'\0' file; do 
+    if file --mime-encoding  "$file" | grep -q iso-8859-1  ;then
+        echo "converting $file from iso-8859-1 to utf-8" 
+        iconv --from-code=iso-8859-1 --to-code=utf-8 "$file" -o "$file"
+   fi
+done
```
Comment (julsemaan) [inline addons/upgrade/to-8.3-conf-latin1-to-utf8.sh:8]: Needs a message saying the job is done so there is output even though no file has been touched

**Comment by julsemaan** [comment]
Also this PR is missing labels and milestone

File: addons/upgrade/to-8.3-conf-latin1-to-utf8.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+find /usr/local/pf/conf -type f -print0 | while read -d $'\0' file; do
```
Comment (julsemaan) [inline addons/upgrade/to-8.3-conf-latin1-to-utf8.sh]: Also, we need to do this for the templates in html/captive-portal/profile-templates

**Comment by jrouzierinverse** [comment]
Done

**Pull request merged**